src/styles/global/_color-classes.scss

```diff
@@ -26,9 +26,6 @@
     @each $mod-name, $mod in $colors-mods {
       &.bv--#{$mod-name} {
         background-color: $mod;
-        @if $mod-name == darken-1 or $mod-name == darken-2 or $mod-name == darken-3 {
```

Comment:
This have to be removed. Now when we add modificator class selector has specificity of 0 2 0 0 and it gest impossible to change text color on such background with bv--text-color classes.

I wonder if we should change approach here and create class like bv--color-red-darken-1 instead of increasing specificity with modificator class.